customer-api/src/Controller/StringManipulationController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Annotation\Route;
+
+class StringManipulationController extends AbstractController
+{
+    /**
+     * @Route("/string-manipulate", name="string_manipulate")
+     */
+    public function index(Request $request): Response
+    {
+        $inputString = $request->query->get('input');
+        $targetString = $request->query->get('target');
+        $result = null;
+
+        if ($inputString !== null && $targetString !== null) {
+            $result = $this->validateStringFromInput($inputString, $targetString) ? 'True' : 'False';
+        }
+
+        return $this->render('string_manipulation/index.html.twig', [
+            'result' => $result,
+        ]);
+    }
+
+
+    public function validateStringFromInput(string $input, string $target): void {
+        $t = intval(trim($target));
+
+        for ($i = 0; $i < $t; $i++) {
+            $s = trim($input);
```

Comment:
use meaningful names -> 's' doesn't say much
also if we only have to trim $input once -> let's keep it out of the loop